index.md

```diff
@@ -2,57 +2,45 @@
 site: sandpaper::sandpaper_site
 ---
 
-This is a new lesson built with [The Carpentries Workbench][workbench]. It is currently in pre-alpha development by the [Edinburgh Parallel Computing Centre][epcc-link], the [National Centre for Atmospheric Science][ncas-link], the [National Oceanography Centre][noc-link], and the [UK Met Office][uk-met-link].
+::: caution
 
-This course is aimed at users and developers who know how to program,
-but have little or no experience in Fortran, and those who may wish to
-have a refresher in Fortran.
+This is a new lesson built with [The Carpentries Workbench][carpentries-workbench].
+It is currently in pre-alpha development by the
+[Edinburgh Parallel Computing Centre][epcc-link],
```

Comment:
I'm at BGS (British Geological Survey) now, although we ought to probably acknowledge the material is partly based on the ARCHER2/EPCC modern fortran material?

Reply:
```suggestion
[British Geological Survey][bgs-link],
```